Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -430,7 +430,7 @@ section
 
 theorem coeff_C_mul_X_pow (x : R) (k n : ℕ) :
     coeff R n (C R x * X ^ k : R⟦X⟧) = if n = k then x else 0 := by
-  simp [X_pow_eq, coeff_monomial]
+  rw [X_pow_eq, coeff_C_mul, coeff_monomial, mul_ite, mul_one, mul_zero]
```

Comment:
This feels less readable to me: without a reason why, I would prefer not to change this.